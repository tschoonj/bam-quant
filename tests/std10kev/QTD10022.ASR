File: C:\xrfdaten\2014KW30\chris\std10kev\std10022.spe
Start: 21.07.2014 15:53:35
Stop: 21.07.2014 15:55:15
BESSY: 298.5059193 298.7038812
IONI: 1.468756E-10 1.465569E-10
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 87606
Run: 100000
Dead: 12
ICR: 37120
OCR: 0
Accumulated BESSY: 59677.4481851
Accumulated Ioni: 2.93168039E-08
N Accumulation: 200
Normfactor:       77871676.
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Ti
exp_x 72.0204010009766 72.0204
exp_y 150.265808105469 150.2658
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 871.55 871.549999496
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 10 10.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 0.03
$PEAKS:   
      1
  22   1      4.509   2754866.      1660.     161.39
