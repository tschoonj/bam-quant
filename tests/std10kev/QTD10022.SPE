$DATA:
0 2000
20493
13135
5612
1689
356
52
8
1
1
1
4
1
0
0
1
4
113
880
970
906
987
867
864
820
787
789
737
700
704
732
720
681
692
652
640
595
632
611
576
608
567
562
557
586
561
539
513
559
570
527
523
498
504
490
541
516
576
523
530
517
516
494
524
491
479
506
471
511
462
507
483
502
480
432
481
477
462
475
461
417
485
441
473
456
485
469
474
448
488
461
472
461
505
446
503
463
485
538
526
512
583
595
557
552
572
532
541
543
556
491
480
490
537
496
509
537
507
486
491
483
530
484
517
545
538
501
544
552
596
707
767
898
1144
1643
2298
2978
3475
3596
3123
2597
1786
1214
950
904
1005
1168
1221
1171
1166
943
770
653
638
723
780
847
929
974
1017
955
840
731
682
656
650
620
580
600
620
641
657
637
664
691
701
746
701
722
775
787
812
798
800
847
821
842
866
873
958
969
969
1082
1051
1027
1107
1060
1110
1098
1058
1060
1052
1065
1117
1179
1199
1239
1407
1438
1615
1727
2067
2327
2822
3517
5107
8555
16888
34755
70054
126019
204396
292976
367065
400589
382577
318658
231764
146465
82335
40438
17968
7561
3494
2145
1989
2267
3176
4847
8127
13782
22443
32967
44112
53135
56436
52491
43531
32119
20554
11826
6030
2578
1093
438
171
80
46
47
38
29
44
30
21
42
29
32
29
32
28
26
24
29
15
17
10
17
16
16
13
16
12
15
9
13
16
13
9
11
15
10
15
12
8
10
11
13
14
4
9
5
7
9
8
5
10
7
6
12
15
12
12
25
35
37
42
54
65
78
65
45
38
27
29
22
15
5
6
6
8
8
5
8
7
1
9
20
11
19
29
31
36
41
68
58
54
47
42
38
33
26
25
16
15
15
12
15
16
23
33
33
22
20
34
47
79
95
136
200
227
256
270
249
232
185
115
80
57
47
21
19
17
18
22
9
17
21
16
15
21
25
33
22
34
25
23
22
22
16
40
23
24
23
24
19
24
24
33
28
43
46
47
35
44
33
30
17
17
10
18
11
10
15
12
17
18
17
20
21
19
28
18
20
13
28
17
30
35
71
88
160
252
327
487
684
788
884
957
937
814
688
557
351
236
141
80
61
46
35
35
44
71
89
125
174
224
239
278
301
262
262
193
176
117
76
65
49
35
33
37
33
62
56
85
98
126
149
217
251
294
333
408
493
525
587
579
556
581
482
455
315
242
215
149
113
83
39
17
9
5
6
1
0
1
0
0
0
1
1
0
1
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
2
1
0
0
0
0
0
1
0
0
0
1
1
0
0
0
0
1
0
0
0
0
0
0
0
0
0
1
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
2
0
0
1
3
1
1
3
1
2
0
1
0
0
0
0
0
0
0
0
0
0
1
0
1
0
0
0
1
1
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
1
1
0
0
0
3
0
0
0
4
2
1
3
0
0
0
0
1
0
0
0
0
1
0
0
0
0
1
1
0
1
0
1
1
0
0
0
1
0
0
0
0
0
1
1
0
2
4
1
0
0
0
2
5
2
3
1
1
2
2
2
2
5
1
0
1
0
0
0
2
0
1
1
1
0
2
1
1
1
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std10kev\std10022.spe
Start: 21.07.2014 15:53:35
Stop: 21.07.2014 15:55:15
BESSY: 298.5059193 298.7038812
IONI: 1.468756E-10 1.465569E-10
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 87606
Run: 100000
Dead: 12
ICR: 37120
OCR: 0
Accumulated BESSY: 59677.4481851
Accumulated Ioni: 2.93168039E-08
N Accumulation: 200
Normfactor:       77871676.
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Ti
 
exp_x 72.0204010009766 72.0204
exp_y 150.265808105469 150.2658
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 871.55 871.549999496
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 10 10.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 0.03
$PEAKS:   
      1
  22   1      4.509   2754866.      1660.     161.39
