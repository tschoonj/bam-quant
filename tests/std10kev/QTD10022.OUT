     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:26:56  บ   
     บ  Spectrum: QTD10022.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 130 - 500        5 iterations done    บ   
     บ  ChiSquare =   276.9   last change =   .04%    lambda= 1.E-07   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -38.2 ๑   100.0   บ     -38.2 ๑      .7   บ   
     บ  GAIN (eV/ch)  บ     20.290 ๑   2.000   บ    20.290 ๑    .003   บ   
     บ det NOISE (eV) บ       34.4 ๑    40.0   บ      34.4 ๑     7.9   บ   
     บ  FANO factor   บ       .173 ๑    .050   บ      .173 ๑    .004   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Ti-Ka                           2754866. ๑   1660.           บ   
     บ    KA1         4.509   1.00000     2754866. ๑   1660.    161.4  บ   
     บ              224.110    133.50         536.           7.84E-01  บ   
     บ                                                                 บ   
     บ  2 Ti-Kb                            408711. ๑    640.           บ   
     บ    KB1         4.932   1.00000      408711. ๑    640.     76.6  บ   
     บ              244.957    139.22         367.           8.30E-01  บ   
     บ                                                                 บ   
     บ  3 Pile_up                           10198. ๑    108.           บ   
     บ    KA1         9.018    .75823        7733. ๑     84.     26.7  บ   
     บ              446.334    185.63         418.           9.69E-01  บ   
     บ    KB1         9.441    .22507        2295. ๑     33.      8.1  บ   
     บ              467.182    189.78         481.           9.73E-01  บ   
     บ    KA1         9.864    .01670         170. ๑     24.    225.6  บ   
     บ              488.029    193.85         593.           9.76E-01  บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
